### ISSUE TITLE
Design - MVP - Inhouse Example

### ISSUE THREAD

**Issue by TheLarkInn** [mutualofomaha/multipage-webpack-plugin#19]
Folder structure will be: 

* A laravel application can be broken up into sections. (Via Composer Packages). These sections live in:

```javascript
`./packages/mutualofomaha/${packageName}`
```

* Each section could have many entry points related to different Server routed views (.twig templates). These JS Entries will be found in:

```javascript
`./packages/mutualofomaha/${packageName}/src/resources/assets/${entryName}/index.js`
```

* The view templates that correspond to these entry points will live in: 

```javascript
`./packages/mutualofomaha/${packageName}/src/resources/views`
```

* Each view template will include a twig partial, generated from this plugin, which contains only the scripts to be loaded for that view. These partials are located in:

```javascript
`./packages/mutualofomaha/${packageName}/src/resources/views/webpack-bundles/${entryName}.bundle.js`
```

**Comment by yarthur** [comment]
Composer packages install in ./vendor/[vendorname]/[packagename]

**Comment by TheLarkInn** [comment]
Thanks I think thats worth noting

**Comment by TheLarkInn** [comment]
@y8r is our moo.com Laravel 5.3 using a custom install location for composer packages?

------



@yarthur is saying its `'./vendor/[vendorname]/[packagename]'`. The sandbox instance we have checked out is `'./packages/[vendorname]/[packagename]'`

**Comment by TheLarkInn** [comment]
Fixed with #20
